azkaban-common/src/main/java/azkaban/jobExecutor/AbstractProcessJob.java

```diff
@@ -36,48 +36,44 @@
 public abstract class AbstractProcessJob extends AbstractJob {
 
   public static final String ENV_PREFIX = "env.";
-  public static final String ENV_PREFIX_UCASE = "ENV.";
   public static final String WORKING_DIR = "working.dir";
   public static final String JOB_PROP_ENV = "JOB_PROP_FILE";
   public static final String JOB_NAME_ENV = "JOB_NAME";
   public static final String JOB_OUTPUT_PROP_FILE = "JOB_OUTPUT_PROP_FILE";
   private static final String SENSITIVE_JOB_PROP_NAME_SUFFIX = "_X";
   private static final String SENSITIVE_JOB_PROP_VALUE_PLACEHOLDER = "[MASKED]";
   private static final String JOB_DUMP_PROPERTIES_IN_LOG = "job.dump.properties";
+
   protected final String _jobPath;
-  private final Logger log;
+  protected String _cwd;
   protected volatile Props jobProps;
   protected volatile Props sysProps;
 
-  protected String _cwd;
-
   private volatile Props generatedProperties;
 
-  protected AbstractProcessJob(final String jobid, final Props sysProps,
-      final Props jobProps, final Logger log) {
+  protected AbstractProcessJob(final String jobid, final Props sysProps, final Props jobProps, final Logger log) {
     super(jobid, log);
 
     this.jobProps = jobProps;
     this.sysProps = sysProps;
     this._cwd = getWorkingDirectory();
     this._jobPath = this._cwd;
-
-    this.log = log;
   }
 
+  /**
+   * TODO: Review to see if this function can be private
```

Comment:
Is this possible to do as part of this PR?

Reply:
sure.